src/hooks/usePostInquiry.ts

```
@@ -0,0 +1,14 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { postInquiry } from '../api/inquiry.api';
+import { AxiosError } from 'axios';
+
+export const usePostInquiry = () => {
+  const queryClient = useQueryClient();
+
+  const mutate = useMutation<void, AxiosError, FormData>({
```

ESLint

[error] 'mutate' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)